quickstart/known-issues.md

```diff
@@ -1,18 +1,25 @@
 # Known Issues
 
-## A Mayastor pod restarts unexpectedly with exit code 132 whilst mounting a PVC
+## Live Issue Tracker
 
-The Mayastor process has been sent the SIGKILL signal as the result of attempting to execute an illegal instruction.  This indicates that the host node's CPU does not satisfy the prerequisite instruction set level for Mayastor \(SSE4.2 on x86-64\).
+Mayastor is currently considered to be beta software.
+
+> "(it) will generally have many more bugs in it than completed software and speed or performance issues, and may still cause crashes or data loss."
+
+The project's maintainers operate a live issue tracking dashboard for defects which they have under active triage and investigation.  It can be accessed [here](https://mayadata.atlassian.net/secure/Dashboard.jspa?selectPageId=10015).  You are strongly encouraged to familisarise yourself with the issues identified there before deploying Mayastor and/or raising further issue reports.
 
-## Mayastor lacks k8s finalizers
+## How is Mayastor Tested?
 
-Finalizers have yet to implemented on key structures, therefore it is important to ensure that the correct tear down sequences are followed, if unintended data loss is to be avoided.
+Mayastor's maintainers perform integration and end-to-end testing on nightly builds and named releases.  Clusters used to perform this testing are composed of worker nodes running Ubuntu 20.04.2 LTS, using the docker runtime 20.10.5 under Kubernetes version 1.19.8.  No other testing configurations are in regular use at this time.
 
-For example, deleting a Mayastor Pool \(MSP\) whilst it is hosting Volume replicas is possible and will result in the faulting of those replicas.  If the replication factor of an implicated volume is equal to one \(i.e. there is _only_ one data replica\), that volume's data will be irrecoverably deleted along with the pool.
+## Common Installation Issues
+
+### A Mayastor pod restarts unexpectedly with exit code 132 whilst mounting a PVC
+
+The Mayastor process has been sent the SIGKILL signal as the result of attempting to execute an illegal instruction.  This indicates that the host node's CPU does not satisfy the prerequisite instruction set level for Mayastor \(SSE4.2 on x86-64\).
```

Comment:
I think it's `SIGILL` not `SIGKILL` https://stackoverflow.com/questions/7901867/what-causes-signal-sigill

Reply:
https://github.com/openebs/mayastor-docs/pull/14